Samples/WinMLSamplesGallery/WinMLSamplesGallery/Samples/AdapterSelection/AdapterSelection.xaml.cs

```diff
@@ -33,6 +33,8 @@ public AdapterSelection()
 
             adapter_options.AddRange(adapters);
             AdapterListView.ItemsSource = adapter_options;
+
+            Task.Run(() => SetModelNameForTelemetry(device));
```

Comment:
why can't you just call setmodelnamefortelemetry directly?

Reply:
It blocks the UI thread and causes the sample to take a while to load since this is called in the constructor. Just added a comment to address this.

Reply:
why is it different only for AdapterSelection page? looks like the rest of them only call SampleBasePage.SetModelNameForTelemetry